README.md

```diff
@@ -160,7 +160,7 @@ the Ticker constructor.
 import requests_cache
 session = requests_cache.CachedSession('yfinance.cache')
 session.headers['User-agent'] = 'my-program/1.0'
-ticker = yf.Ticker('msft aapl goog', session=session)
+ticker = yf.Ticker('msft', session=session)
```

Comment:
Instead, replace with Tickers(), I think that was original intent

Reply:
`AttributeError: 'Tickers' object has no attribute 'actions'`

Reply:
Good point, `Ticker()` is correct solution